.github/workflows/sync-with-upstream.yaml

```diff
@@ -0,0 +1,74 @@
+name: 'Sync with Upstream'
```

Comment:
this as well https://github.com/kumahq/go-control-plane/blob/main/.github/workflows/sync-with-upstream.yaml 🤷 are you on an outdated branch? can you update?